app/src/renderer/vuex/modules/send.js

```diff
@@ -37,13 +38,13 @@ export default ({ node }) => {
     // extract "to" address
     let to = args.to
     delete args.to
-    args.gas = `50000000`
 
     // submit to LCD to build, sign, and broadcast
     let req = to ? node[type](to, args) : node[type](args)
 
     let res = await req.catch(err => {
       let message
+      console.log(err)
```

Comment:
Not sure if it's a leftover or is here on pourpose

Reply:
good catch, fixed